data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/jsonview/StudentView.java

```diff
@@ -0,0 +1,100 @@
+package io.micronaut.data.jdbc.oraclexe.jsonview;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.micronaut.data.annotation.GeneratedValue;
+import io.micronaut.data.annotation.Id;
+import io.micronaut.data.annotation.JsonView;
+import io.micronaut.data.annotation.MappedEntity;
+import io.micronaut.data.annotation.Relation;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@JsonView
+@MappedEntity(value = "STUDENT_VIEW")
```

Comment:
can we combine these into one annotation? So I can just declare:

```java
@JsonView("STUDENT_VIEW")
```

Also can we just declare `@JsonView` and by convention default to `STUDENT_VIEW` based on the naming strategy?

Reply:
Just to ask, does that mean some visitor is needed so when JsonView is found then need to annotate with MappedEntity? We might need also alias (can be useful), schema etc. basically duplicate all what we have in MappedEntity?

Reply:
Ok, I have done it using `class JsonViewMapper implements TypedAnnotationMapper<JsonView>`. Not sure if the best approach but it does the job.

Reply:
might be possible to just meta annotate (ie annotate `@JsonView` annotation with the appropriate `@EntityRepresentation` annotation)

Reply:
There is this value that we read from `JsonView` so I thought we should use mapper. Not sure.

```
/**
     * The name of the single column in the view.
     *
     * @return the column name (default DATA)
     */
    String column() default DEFAULT_COLUMN_NAME;
```

although most likely will be `DATA` value always.